src/header/header.scss

```diff
@@ -31,6 +31,12 @@
     flex: 2 1 33%;
     @include iui-header-buttons;
 
+    > nav {
+      display: flex;
+      align-items: center;
```

Comment:
Why it is needed?

Reply:
It's either this or I need `display: contents` (which is not supported in IE). Otherwise it breaks styling